tee-worker/app-libs/stf/src/getter.rs

```diff
@@ -58,7 +58,7 @@ impl From<TrustedGetterSigned> for Getter {
 #[allow(non_camel_case_types)]
 pub enum PublicGetter {
 	some_value,
-	nonce(AccountId),
+	nonce(AccountId, u32),
```

Comment:
In my opinion the getter should determine the number of pending transactions instead of relying on provided value, which also requires callers to somehow determine it.

If it stays like this, please mark PR as breaking change.